lib/gui/app.js

```diff
@@ -94,13 +94,14 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
   const currentVersion = packageJSON.version;
   const currentReleaseType = release.getReleaseType(currentVersion);
   const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates({
-    ignoreSleepUpdateCheck: currentReleaseType !== release.RELEASE_TYPE.PRODUCTION
+    releaseType: currentReleaseType,
+    lastSleptUpdateNotifier: settings.get('lastSleptUpdateNotifier'),
+    lastSleptUpdateNotifierVersion: settings.get('lastSleptUpdateNotifierVersion') || packageJSON.version
```

Comment:
I wonder if it might make sense to invert the logic, and always pass in the `packageJSON.version` here, and then have `.shouldCheckForUpdates` check that against `settings.get('lastSleptUpdateNotifierVersion')` ?

And then potentially `.shouldCheckForUpdates` could itself also get the `releaseType` from the version string?